api/Window.json

```
@@ -2338,24 +2338,26 @@
       "gamepadconnected_event": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/gamepadconnected_event",
-          "spec_url": "https://w3c.github.io/gamepad/#the-gamepadconnected-event",
+          "spec_url": [
+            "https://w3c.github.io/gamepad/#the-gamepadconnected-event",
+            "https://w3c.github.io/gamepad/#event-gamepadconnected"
+          ],
```

These two URLs go to the same section. Let's keep just the one that's used in spec's own navigation:

```suggestion
          "spec_url": "https://w3c.github.io/gamepad/#event-gamepadconnected",
```